### DISCUSSION TITLE
Why use TYPO3 Surf versus GitHub actions with FTP?

### DISCUSSION THREAD

**Discussion by quantuminformation** [TYPO3/Surf#799] [Q&A] [answered]
I am new to typo but I have some client work we have about 10 TYPO3 projects.

**Comment by simonschaufi** [comment, marked as answer]
Welcome to the TYPO3 community!

TYPO3 Surf does a lot more than just copy files to a server. It creates a new folder for each release. That has the advantage that the website won't be broken during the time of deployment and will be switched to the new folder once it's done.

Then TYPO3 Surf also migrates the DB and clears the caches.

If you want, you can add more custom build steps like compiling scss to CSS or whatever you need.

I hope this helps.

**Comment by quantuminformation** [comment]
ah great thanks
